docs/dev/topology-over-raft.md

```diff
@@ -16,12 +16,16 @@ Node state can be one of those:
 Nodes in state left are never removed from the state.
 
 State transition diagram for nodes:
-```
-{none} ------> {bootstrapping|replacing} ------> {normal} <---> {rebuilding}
- |                   |                              |
- |                   |                              |
- |                   V                              V
- ----------------> {left}  <--------  {decommissioning|removing}
+```mermaid
+stateDiagram-v2
+    [*] --> bootstrapping|replacing
+    [*] --> [*]
+    bootstrapping|replacing --> normal
+    bootstrapping|replacing --> [*]
+    normal --> rebuilding
+    normal --> decommissioning|removing
+    rebuilding --> normal
+    decommissioning|removing --> [*]
```

Comment:
This should be "left", no? also "none" is not presented.

Reply:
they are practically the "start" and "end" state. so I intentionally merge them into the start and end respectively. If you still think we should have them explicitly in the diagram, I will add them back.

Reply:
i changed back to the named state. this preserves the naming of the original ascii-based diagram.